### PR TITLE
[MIG] portal_payment_mode: Migration to 9.0

### PR DESCRIPTION
**Pull request by chienandalu**
Portal Payment Mode
===================

This module extends the functionality of *account_payment_partner* module to
allow to portal users view theirs invoices adding security permissions to
payment mode model.

cc @Tecnativa

### CODE REVIEW

File: portal_payment_mode/__openerp__.py [hunk 1]
```diff
@@ -17,10 +17,11 @@
     ],
     'author': 'Antiun Ingeniería S.L., '
```
Comment (pedrobaeza) [inline portal_payment_mode/__openerp__.py]: Please remove old brands

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
And check Travis status

**Comment by chienandalu** [comment]
@pedrobaeza Changes done

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@luismontalba please review

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Pull request merged**